### ISSUE TITLE
Could not find method dexcount() for arguments 

### ISSUE THREAD

**Issue by ParkerK** [KeepSafe/dexcount-gradle-plugin#189]
When I add dexcount {...} to my app gradle, I get this error:

`Error:org.gradle.internal.metaobject.AbstractDynamicObject$CustomMessageMissingMethodException: Could not find method dexcount() for arguments [build_co6u1j4pjvt1t3moujul16v4c$_run_closure1@22294fa4] on project ':app' of type org.gradle.api.Project.`

If I remove the `dexcount` block then I can build and run the app just fine (it just doesn't give any dex counts in the output folder). 

app gradle file is:

```
buildscript {
    repositories {
        mavenCentral() // or jcenter()
    }

    dependencies {
        classpath 'com.getkeepsafe.dexcount:dexcount-gradle-plugin:0.7.2'
    }
}

// make sure this line comes *after* you apply the Android plugin

apply plugin: 'com.android.application'
apply plugin: 'kotlin-android-extensions'
apply plugin: 'kotlin-android'
apply plugin: 'com.getkeepsafe.dexcount'

//apply plugin: 'kotlin-kapt'
dexcount {
    format = "list"
    includeClasses = false
    includeClassCount = false
    includeFieldCount = true
    includeTotalMethodCount = false
    orderByMethodCount = false
    verbose = false
    maxTreeDepth = Integer.MAX_VALUE
    teamCityIntegration = false
    teamCitySlug = null
    runOnEachPackage = true
    maxMethodCount = 64000
}

android {
    signingConfigs {
        config {
           ...
        }
    }
    compileSdkVersion 25
    buildToolsVersion "26.0.0"
    defaultConfig {
        applicationId "com.example.app"
        minSdkVersion 23
        targetSdkVersion 25
        versionCode 1
        versionName "1.0"
        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
        signingConfig signingConfigs.config
    }
    buildTypes {
        release {
            minifyEnabled true
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
        }
        debug {
            versionNameSuffix "-debug"
        }
    }
    flavorDimensions "mode"
    productFlavors {
        free {
            dimension "mode"
        }
        pro {
            dimension "mode"
            applicationIdSuffix ".pro"
        }
    }
}


dependencies {
   ...
}

apply plugin: 'com.google.gms.google-services'


```

Thanks in advance for any guidance!

**Comment by benjamin-bader** [comment]
Certainly seems like a bug on our end.  What version of the Android Gradle Plugin are you using?  What version of Gradle?

**Comment by ParkerK** [comment]
Ah, if that's the case it may another AS3 Canary break. I'm on Studio 3 Canary 6 using gradle 3.0.0-alpha plugin and gradle itself is 4.1-milestone-1

On a side not, hitting the 'Run app' (green button) should be the same as `gradlew --build` right?

**Comment by benjamin-bader** [comment]
I missed the part where you said that, without the `dexcount` block, your app builds **but no method counts are done**.  That suggests to me that you have Instant Run enabled.  True?

**Comment by ParkerK** [comment]
I do have instant run enabled

**Comment by benjamin-bader** [comment]
OK - would you please turn it off, put the `dexcount` config block back into your gradle file, then build?  If the build succeeds, then I'll be confident that I know the bug.

**Comment by ParkerK** [comment]
It now builds without an error, but I'm still not seeing output. Hopefully that is what you are expecting :)

**Comment by benjamin-bader** [comment]
Great, thank you - I'll have a fix shortly.  Appreciate your help!

**Comment by benjamin-bader** [comment]
I've just released version 0.7.3 to Maven Central, which contains this fix.  It should be available within the next 15 minutes.
